app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -261,43 +277,78 @@ class AddDeadlineActivity : AppCompatActivity() {
         if (titleDeadline == "") {
             Toast.makeText(this, getString(R.string.enter_a_title), Toast.LENGTH_SHORT).show()
         } else {
-            // Add the deadline
-            val deadline = Deadline(
-                titleDeadline,
-                DeadlineState.TODO,
-                selectedDate,
-                textDescription.text.toString()
-            )
-
-            val notificationsTimes = retrieveNotificationsTimes()
-
-            Toast.makeText(this, getString(R.string.deadline_created), Toast.LENGTH_SHORT).show()
-
             // Reset the text input for future use
             textTitle.text = ""
-
-            deadlineListViewModel.addDeadline(deadline) {
-                DeadlineNotification.editNotification(it, deadline, notificationsTimes, this)
-                finish()
+            //loading bar
+            progressBar.visibility = View.VISIBLE
+            //start upload
+            uploadPdfToFirebase(pdfData) { ref ->
+                //hide loading bar
+                progressBar.visibility = View.GONE;
+
+                // create the deadline
```

Comment:
Since you will have new commits to do, try to unify the style of the comments among the file. The format that we used is:
// Uppercase first